tcms/package.json

```diff
@@ -2,6 +2,8 @@
   "dependencies": {
     "bootstrap-switch": "3.3.4",
     "marked": "0.8.2",
+    "moment": "2.24.0",
+    "moment-timezone": "0.5.28",
```

Comment:
These are dependecies to patternfly you should not list them explicitly, especially with version differences.

Reply:
what if patternfly removes them from their dependency tree? 

it seems better to list everything we depend on and not assume the implicit existence of some dependency - https://12factor.net/dependencies

Reply:
> what if patternfly removes them from their dependency tree?

That's why we have `tcms.tests.storage.RaiseWhenFileNotFound` during tests